chaoscenter/web/src/strings/strings.en.yaml

```
@@ -1076,6 +1087,7 @@ tuneFaultProperties: Tune fault properties
 tuneFaultWeight: Tune fault weightage
 tuneWeight: Tune Weight
 type: Type
+typeofdata: Type of data
 udpatePassword: Update Password
```

```suggestion
updatePassword: Update Password
```

can you change it too with relevent files?